packages/client/test/sync/fullsync.spec.ts

```diff
@@ -109,8 +109,9 @@ describe('[FullSynchronizer]', async () => {
       txPool,
       execution,
     })
-    ;(sync as any).pool.open = vi.fn().mockResolvedValue(null)
-    ;(sync as any).pool.peers = []
+    sync['pool'].open = vi.fn().mockResolvedValue(null)
+    /// @ts-expect-error -- Assigning simpler config for testing
```

Comment:
Small question (might be answered elsewhere): why these three-`/` comments?

Reply:
Good question! It is for the typescript compiler, see here: 

> Triple-slash directives are special comments that provide instructions to the compiler about how to process a file. These directives begin with three consecutive slashes (///) and are typically placed at the top of a TypeScript file and have no effects on the runtime behavior.

https://gibbok.github.io/typescript-book/book/triple-slash-directives/#:~:text=Triple%2Dslash%20directives%20are%20special,effects%20on%20the%20runtime%20behavior.